src/OrchardCore.Modules/OrchardCore.ContentTypes/Controllers/AdminController.cs

```diff
@@ -170,9 +170,9 @@ public async Task<ActionResult> EditPOST(string id, EditTypeViewModel viewModel)
             viewModel.DisplayName = contentTypeDefinition.DisplayName;
             viewModel.Editor = await _contentDefinitionDisplayManager.UpdateTypeEditorAsync(contentTypeDefinition, this);
 
-            if(IsAlphaNumeric(viewModel.Settings["ContentTypeSettings"].Value<string>("Stereotype")))
+            if(!IsAlphaNumeric(viewModel.Settings["ContentTypeSettings"].Value<string>("Stereotype")))
```

Comment:
It;s the wrong place to do the validation. This should be done in `ContentTypeSettingsDisplayDriver`

Reply:
I was thinking about that at the beginning

Reply:
BTW how to use the `ModelState` there to add an error

Reply:
the `Updater` property has the model state. You'll see when you are in the class. There are many examples of validation in display drivers.

Reply:
Thanks